decide/lic0.test.cpp

```diff
@@ -2,10 +2,19 @@
 #include <decide/decide.hpp>
 #include <decide/lics.hpp>
 
```

Comment:
`cassert` is not included.

```suggestion
#include <cassert>

```